.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: Stale PR Reminder
+
+on:
+  schedule:
+    - cron: '0 12 * * *' # Run every day at 12:00 UTC
+
+jobs:
+  stale-reminder:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Determine staleness
+      uses: JasonEtco/check-if-merged@v1
+      with:
+        stale-phrase: 'inactive'
+        days-before-stale: 7
```

Comment:
The OFEP process can take a while. I would recommend settings these values fairly high. Perhaps something like this:

days-before-stale: 30
days-before-close: 90